shared/img/screen.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="15px" height="14px" viewBox="0 0 15 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 3.7.2 (28276) - http://www.bohemiancoding.com/sketch -->
+    <title>Screen Share Active</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Room-Description---Hello-Window" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Room---Screen-Sharing-Enabled" transform="translate(-1319.000000, -143.000000)" fill="#4C4C4C">
```

Comment:
@Mardak these images come from Pau. I don't think the color of the icons must be a blocker since we haven't the visuals yet, and probably we should change colors and the visual aspect of lot of components/images.

I'm going to svgo the files.